proto/cmp/types/v3/payment.proto

```diff
@@ -0,0 +1,25 @@
+syntax = "proto3";
+
+package cmp.types.v3;
+
+import "cmp/types/v1/credit_card.proto";
+
+// Additional payment info message with currently only a single field of
+// `CreditCard` message type.
+//
+// ![Diagram](https://storage.googleapis.com/docs-cmp-files/diagrams/proto/cmp/types/v3/payment.proto.dot.xs.svg)
+//
+// [Open Message Diagram](https://storage.googleapis.com/docs-cmp-files/diagrams/proto/cmp/types/v3/payment.proto.dot.svg)
+message AdditionalPaymentInfo {
+  cmp.types.v1.CreditCard credit_card = 1;
+}
+
+// Payment Types
+enum PaymentType {
+  PAYMENT_TYPE_UNSPECIFIED = 0;
+  PAYMENT_TYPE_OTHER = 1;
+  PAYMENT_TYPE_CRYPTO = 2;
+  PAYMENT_TYPE_BANK_TRANSFER = 3;
+  PAYMENT_TYPE_CREDITCARD = 4;
+  PAYMENT_TYPE_PAYPAL = 5;
+}
```

Comment:
Visa just released a creditcard with crypto currencies: https://cointelegraph.com/news/avalanche-visa-crypto-spending-card. Is our structure OK (creditcard+crypto)??

Reply:
I assume that Visa will convert the payment to whatever currency is used the the seller, no?

Reply:
maybe in the beginning, but that does not matter for us, we have a user that pays with crypto using Visa
I can imagine circulation of stable-coin type of currencies. Then the concept of CreditCard never goes with Crypto will not hold.